src/client/context.ts

```diff
@@ -35,22 +36,60 @@ export interface Context {
     id?: {[key: string]: string|undefined;};
 
     /**
+     * @hidden
      * A context object is open for extension with any custom properties/metadata.
      */
     [key: string]: unknown;
 }
 
-// Built-in contexts
+/**
+ * Built-in context to define a contact.
+ */
 export interface ContactContext extends Context {
+    /**
+     * The context type is always 'fdc3.contact'.
+     */
     type: 'fdc3.contact';
+    /**
+     * Name of the contact.
+     */
     name: string;
+    /**
+     * The contact data. Currently has optional fields for email address, Twitter handle, and phone number.
+     */
     id: {[key: string]: string}&{email?: string; twitter?: string; phone?: string};
 }
-export interface SecurityContext extends Context {
-    type: 'fdc3.security';
-    id: {[key: string]: string}&{default: string};
+
+/**
+ * Built-in context to define a financial instrument.
+ */
+export interface InstrumentContext extends Context {
+    /**
+     * The context type is always 'fdc3.instrument'.
+     */
+    type: 'fdc3.instrument';
+    /**
+     * Can contain some of a ticker, an [ISIN](https://www.isin.org/isin/), a [CUSIP](https://www.cusip.com/cusip/index.htm),
+     * a [SEDOL](https://www.londonstockexchange.com/products-and-services/reference-data/sedol-master-file/sedol-master-file.htm),
+     * a [Reuters Instrument Code (RIC)](https://en.wikipedia.org/wiki/Reuters_Instrument_Code),
+     * a [Bloomberg Ticker](https://www.bloomberg.com/professional/product/market-data/),
+     * a [PERMID](https://permid.org/), and a [FIGI](https://www.openfigi.com/about/figi).
+     */
+    id: {[key: string]: string}&{ticker?: string; ISIN?: string; CUSIP?: string; SEDOL?: string;
+        RIC?: string; BBG?: string; PERMID?: string; FIGI?: string;};
 }
+
+/**
+ * Built-in context to define an organization.
+ */
 export interface OrganizationContext extends Context {
+    /**
```

Comment:
As with InstrumentContext, leaving `name` out makes it inherit the comment from the parent. Want to be sure that's what we want.

Reply:
Again, I think this is correct. Better to have the LEI or whatever than just free text name strings, so we should be relying on them.

Reply:
Again, more about the wording of the doc comment on the derived type than the existence of the name property itself.

Reply:
Got it, fixing